src/preprocess/preprocess.go

```diff
@@ -369,6 +369,11 @@ func exposeAPIs_main(input string) string {
 		`\(\({[^}]*,\s*imageSrc`,
 		`Spicetify.Snackbar.enqueueImageSnackbar=${0}`)
 
+	// Menu hook
+	utils.Replace(&input, `children:([\w_$]+),onClose:[\w_$]+,getInitialFocusElement:[\w_$]+,onFocusVerticalItem:[\w_$]+.+?\}\)=>\{`, `${0}${1}=[Spicetify.ContextMenuV2.renderItems(),${1}].flat();`)
+
+	utils.Replace(&input, `\(0,([\w_$]+)\.jsx\)\([\w_$]+\.[\w_$]+,\{value:"contextmenu"[^\}]+\}\)\}\)`, `${1}.jsx((Spicetify.ContextMenuV2._context||(Spicetify.ContextMenuV2._context=i.createContext(null))).Provider,{value:{props:e?.props,trigger:l,target:p},children:${0}})`)
```

Comment:
**i**.createContext, **e**?.props, trigger:**l**, target:**p** ?

Reply:
This is now resolved, as I'm using regex to find those variable names.